run_echo_air.py

```diff
@@ -226,6 +226,13 @@
         conc.output_concentrations(output_region, output_dir, f_out, ca_shp_path, shape_out)
         logging.info("- [CONCENTRATION] Concentration files output into: {}.".format(output_dir))
 
+        # Find maximum emitting area
```

Comment:
I think this isn't being used anymore, right? If so, we should delete this whole chunk